src/io/flutter/actions/FlutterSdkAction.java

```diff
@@ -49,15 +50,25 @@
     return pubspec == null ? null : Pair.create(module, pubspec);
   }
 
-  protected static VirtualFile findPubspecFrom(Project project, PsiFile psiFile) {
+  protected static VirtualFile findPubspecFrom(@NotNull Project project, PsiFile psiFile) {
     if (psiFile == null) {
-      return null;
+      return findPubspecFrom(ModuleManager.getInstance(project).getModules());
     }
     final VirtualFile file = psiFile.getVirtualFile();
     final VirtualFile contentRoot = ProjectRootManager.getInstance(project).getFileIndex().getContentRootForFile(file);
     return contentRoot == null ? null : contentRoot.findChild(FlutterConstants.PUBSPEC_YAML);
   }
 
+  private static VirtualFile findPubspecFrom(@NotNull Module[] modules) {
+    for (Module module : modules) {
+      final VirtualFile file = findPubspecFrom(module);
```

Comment:
My thinking was to return the first pubspec we find on the assumption that there should not likely be more than one (or that if there were it wouldn't matter).  Not sure if that's the right solution though TBH.